### PR TITLE
finished tests

### PR DESCRIPTION
**Pull request by Archethect**
This PR add tests for each event handler

### CODE REVIEW

File: license-sale-arb-sepolia/src/play-fi-license-sale.ts [hunk 1]
```diff
@@ -8,80 +8,81 @@ import {
...
   //Initialize stats
-  let stat = new Stat(Bytes.fromI32(0));
+  let stat = new Stat(BigInt.zero().toString());
   stat.totalClaims = BigInt.zero();
   stat.totalProceeds = BigInt.zero();
   stat.save()
 
   //Set team phase paused
-  let teamPhase = new Phase(Bytes.fromI32(0));
+  let teamPhase = new Phase(BigInt.zero().toString());
```
Comment (therealdannzor) [inline license-sale-arb-sepolia/src/play-fi-license-sale.ts]: q: would it make more sense to set an id number to another value than the init value?

**Review by therealdannzor — COMMENTED** [review]
looks fine, just a minor question

**Review by eludius18 — DISMISSED** [review]
LGTM ! 👍

File: license-sale-arb-sepolia/src/play-fi-license-sale.ts (hunk 1 above)
Comment (Archethect) [reply license-sale-arb-sepolia/src/play-fi-license-sale.ts]: There is no default value in subgraph. It either exists or it doesn't. It's not like in the EVM where the default values actually exist.

**Review by therealdannzor — DISMISSED** [review]

**Review by therealdannzor — APPROVED** [review]
LGTM

**Pull request merged**